### PR TITLE
WFNEWS-1076 : Add wfone-notifications-api to terraform

### PR DESCRIPTION
**Pull request by sukhpalp**

### CODE REVIEW

File: terraform/alb.tf [hunk 1]
```diff
@@ -400,6 +423,28 @@ resource "aws_lb_listener_rule" "wfnews_host_based_weighted_routing_wfss_pointid
...
+  listener_arn = data.aws_alb_listener.wfnews_server_front_end.arn
+
+  action {
+    type             = "forward"
+    target_group_arn = aws_alb_target_group.wfone-notifications-api.arn
+  }
+
+  condition {
+    host_header {
+      values = [for sn in var.pointid_names : "${sn}.*"]
```
Comment (vivid-cpreston) [inline terraform/alb.tf:437]: This should be var.wfone_notifications_api_names

File: terraform/ecs.tf [hunk 2]
```diff
@@ -969,7 +969,281 @@ resource "aws_ecs_task_definition" "wfnews_apisix" {
...
+          awslogs-group         = "/ecs/${var.pointid_names[0]}"
+          awslogs-region        = var.aws_region
+          awslogs-stream-prefix = "ecs"
+        }
+      }
+    }
+  ])
+}
+
+resource "aws_ecs_task_definition" "wfone_notifications_api" {
```
Comment (vivid-cpreston) [inline terraform/ecs.tf:1128]: Did you intend for this to be commented out for now?

**Review by vivid-cpreston — CHANGES_REQUESTED** [review]
Looks good, just a couple things

**Review by dhlevi — APPROVED** [review]
Other changes look good. Unsure about the status of the commented out piece so just make sure you're coordinating with @vivid-cpreston

File: terraform/variables.tf [hunk 3]
```diff
@@ -702,6 +726,12 @@ variable "pointid_container_name" {
   type        = string
 }
 
+variable "wfone_notifications_api_container_name" {
+  description = "Name of DB container"
```
Comment (dhlevi) [inline terraform/variables.tf:730]: Minor note, but I assume this description is incorrect for the API container

**Review by dhlevi — APPROVED** [review]

**Comment by dhlevi** [comment]
@vivid-cpreston

**Review by vivid-cpreston — APPROVED** [review]
Looks good!

**Pull request merged**